tap_github/schemas/review_comments.json

```diff
@@ -26,6 +26,97 @@
     "node_id": {
       "type": ["null", "string"]
     },
+    "_links": {
+      "type": ["null", "object"],
+      "properties": {
+        "self": {
+          "type": ["null", "object"],
+          "properties": {
+            "href": {
+              "type": ["null", "string"]
+            }
+          }
+        },
+        "html": {
+          "type": ["null", "object"],
+          "properties": {
+            "href": {
+              "type": ["null", "string"]
+            }
+          }
+        },
+        "pull_request": {
+          "type": ["null", "object"],
+          "properties": {
+            "href": {
+              "type": ["null", "string"]
+            }
+          }
+        }
+      }
+    },
+    "url": {
+      "type": ["null", "string"]
+    },
+    "original_start_line": {
+      "type": ["null", "integer"]
+    },
+    "start_side": {
+      "type": ["null", "string"]
+    },
+    "body_text": {
+      "type": ["null", "string"]
+    },
+    "original_line": {
+      "type": ["null", "integer"]
+    },
+    "reactions": {
```

Comment:
Can't we use refrence of reactions?

Reply:
Used reference of reactions

Reply:
I can still see the object instead of refrence.

Reply:
Not sure, can you refresh, because I've already used ref